packages/snap/snap.manifest.json

```diff
@@ -1,13 +1,13 @@
 {
-  "version": "1.2.2",
+  "version": "0.2.1",
```

Comment:
@neekolas not sure why this changed

Reply:
That's odd. It gets copied from the package.json, which gets automatically updated by semantic-release during publishing. So, I'm not sure how much it matters since the publish flow should be:
1. Semantic release updates the package.json to the latest version
2. Semantic release rebuilds the Snap package, which will update this file
3. The final output gets uploaded to NPM